woocommerce/woocommerce-android
# Stripe SDK version update

Closes: #7175


### Description


@malinajirka @AnirudhBhat 
I am a bit hesitant if we should merge it:
* I don't see any vital for us changes
* Somehow this change affects our compilation and I could not figure out what exactly causes that

On other hand, updating dependency more often is easy than shelving that for a while.

Wdyt?

Changes:

### 2.12.0 - 2022-07-25
Update: @OnReaderTips annotation is no longer required for using the on-reader tipping feature
New: Added autoReconnectOnUnexpectedDisconnect & bluetoothReaderReconnectionListener to the [BluetoothConnectionConfiguration](https://stripe.dev/stripe-terminal-android/external/com.stripe.stripeterminal.external.models/-connection-configuration/-bluetooth-connection-configuration/index.html). When enabled, the SDK will attempt to restore connection upon any unexpected disconnect to a reader. See [Stripe Doc](https://stripe.com/docs/terminal/payments/connect-reader?terminal-sdk-platform=android&reader-type=bluetooth#automatic-reconnection) for details.
### 2.11.0 - 2022-06-27
Beta: Add ability to specify capture method when creating a PaymentIntent via the SDK.
### 2.10.2 - 2022-06-15
Fix: Fixed a crash for devices running Android 7.1 and lower. See https://github.com/stripe/stripe-terminal-android/issues/234 for details.

### Testing instructions

* Run IPP with a real reader and a simulator
* Notice that nothing has changed
